include/google/protobuf/any.proto

```diff
@@ -0,0 +1,155 @@
+// Protocol Buffers - Google's data interchange format
```

Comment:
Do I get correct that we do not download proto files during protoc wrapper build, but use these vendored?

Reply:
from what i understand, we use the vendored files and then embed them into the code. this makes sense because each of our releases has a paired upstream-protoc dependency anyway